CMakeLists.txt

```diff
@@ -73,6 +75,9 @@ add_message_files(
         FILES
         Course.msg
         Waypoint.msg
+        DriveVelCmd.msg
+        RAOpenLoopCmd.msg
```

Comment:
Ok now that we're changing to ROS, I think it might be worthwhile just sticking to using [sensor_msgs/JointState ](http://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/JointState.html) and just publishing to topics (e.g. ra/open_loop/joint_a and ra/open_loop/joint_d and ra/open_loop/gripper). This way we won't have to use RAOpenLoopCmd and HandCmd. Maybe @niwhsa9 has thoughts on if this is better, but I think it definitely is.

Reply:
I agree that does seem like a better solution, would we just only use the velocity array?

Reply:
Yes, we would probably only use the velocity array and leave the rest blank. We could also use the JointState msg and only publish to one topic instead of two separate topics for open loop vs closed loop. Likewise, ESW should use the JointState msg to send data back (treating effort as torque I assume).